homeassistant/components/light/mqtt.py

```diff
@@ -87,6 +105,14 @@
     vol.Optional(CONF_RGB_STATE_TOPIC): mqtt.valid_subscribe_topic,
     vol.Optional(CONF_RGB_VALUE_TEMPLATE): cv.template,
     vol.Optional(CONF_STATE_VALUE_TEMPLATE): cv.template,
+    vol.Optional(CONF_HUE_COMMAND_TOPIC): mqtt.valid_publish_topic,
+    vol.Optional(CONF_HUE_STATE_TOPIC): mqtt.valid_subscribe_topic,
+    vol.Optional(CONF_HUE_VALUE_TEMPLATE): cv.template,
+    vol.Optional(CONF_HUE_SCALE, default=DEFAULT_HUE_SCALE):
+        vol.All(vol.Coerce(int), vol.Range(min=1)),
```

Comment:
Do we really need a hue scale? I mean users can easily convert the hue from their scale of choice to the internal 0-360 range using templates.

Also, having the hue scale be an integer doesn't make much sense to me because maybe some people want the range 0-2π

Reply:
I will remove the scale and add a command template.